source/_integrations/knx.markdown

```diff
@@ -951,6 +951,29 @@ entity_category:
   required: false
   type: string
   default: None
+fan_speed_address:
+  description: KNX group address for setting the percentage or step of the fan. *DPT 5.001* or *DPT 5.010*
+  required: false
+  type: [string, list]
+fan_speed_state_address:
+  description: KNX group address for retrieving the percentage or step of the fan. *DPT 5.001* or *DPT 5.010*
+  required: false
+  type: [string, list]
+fan_max_step:
+  description: The maximum amount of steps for the fan.
+  required: false
+  type: integer
+  default: 3
+fan_speed_mode:
+  description: Fan speed group address data type. `percent` for *DPT 5.001* and `step` for *DPT 5.010*.
+  required: false
+  type: string
+  default: percent
+fan_zero_mode:
+  description: The fan mode for the zero speed, either `off` or `auto`. This affects the fan modes displayed in the UI.
+  required: false
+  type: string
+  default: off
```

Comment:
```suggestion
  default: "off"
```
Not sure if this works, but for some reason, `off` renders "false" here.
<img width="408" alt="Bildschirmfoto 2024-09-24 um 21 24 06" src="https://github.com/user-attachments/assets/e8811c96-ba19-490d-a261-8ff9ace54910">

Reply:
looks good now